.github/workflows/tests.yml

```diff
@@ -66,6 +66,7 @@ jobs:
         # TODO(hvy): Fix samplers_tests/test_samplers.py to not require optional dependencies and remove these installs.
         pip install scikit-optimize
         pip install cma
+        pip install botorch torch==1.8.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Why is fixing the version here required?

Reply:
Sorry for the delayed response. I followed the version constraint in `setup.py`.

https://github.com/optuna/optuna/blob/7c889c4a8c46cf8c8adc5a967472c0b4900bb428/setup.py#L163